packages/flutter_test/lib/src/controller.dart

```diff
@@ -252,9 +253,12 @@ abstract class WidgetController {
   /// the given widget, assuming it is exposed.
   ///
   /// If the center of the widget is not exposed, this might send events to
```

Comment:
This paragraph could be a macro since its repeated so often?

Reply:
done